python/lsst/afw/geom/skyWcs/skyWcsContinued.py

```diff
@@ -20,9 +20,80 @@
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
 
-__all__ = []
+import numpy as np
 
+from lsst.utils import continueClass
 from ..python import reduceTransform
 from .skyWcs import SkyWcs
 
+__all__ = []
+
+
+@continueClass  # noqa: F811
+class SkyWcs:
+    def pixelToSkyArray(self, x, y, degrees=False):
+        """
+        Convert numpy array pixels (x/y) to numpy array sky (ra/dec)
```

Comment:
It'd be clearer to use (x,y) instead of (x/y) and similarly for (ra/dec). Similarly for the inverse function.